lib/validators/url_validator.rb

```diff
@@ -24,16 +24,51 @@ def validate_each(record, attribute, value)
         end
 
         record.errors[attribute] << options[:message] unless value =~ regexp
```

Comment:
As mentioned in the last review, maybe it's better to `return` early when `value` !~ `regexp`?  Since otherwise the same `options[:message]` could potentially be inserted more than once.

Or perhaps this `regexp` check could become an individual `def validate_regexp`, which is then checked in the `catch(:invalid)` block below?

What are your thoughts on this?